client/src/components/VerbContainer.jsx

```diff
@@ -62,89 +160,104 @@ export default class VerbContainer extends Component {
+        const newState = { tenses };
+         localStorage.setItem(
+           'tenses',
+           JSON.stringify(tenses)
+        );
+         return newState;
+      });
+      
+    } else {
+      this.setState((prevState) => {
+        const newState = {
+          tenses: {
+            ...prevState.tenses,
+            [tenseNames]: !prevState.tenses[tenseNames]
+          }
+         }
+         localStorage.setItem(
+           'tenses',
+           JSON.stringify(Object.assign(prevState.tenses, newState.tenses))
+        );
+         return newState;
+      });
+    }
   }
 
-  submitVerbs(formValues) {
-    this.setState({ formValuess });
+  submitVerbs(values) {
+    this.setState({ formValues: values });
   }
 
   render() {
     const { hasLoaded, tenses, verbs, idx } = this.state;
     const verb = verbs[idx];
 
     return (
-      <div style={{ width: '100%' }}>
-      <div style={{ display: 'flex' }}>
-        <TenseHeader activeTenses={ tenses } toggleTense={ this._toggleTense } />
-      </div>
-      {
-        !hasLoaded ?
-        <h1>Loading Some Verbs</h1> :
-        <div>
-        <div style={{ display: 'flex', alignItems: 'center' }}>
-          <h1>{ verb.infinitive }</h1>
-          <p style={{ marginLeft: '20px' }}>({ verb.infinitiveEnglish })</p>
-          <p
-            style={{
-              border: '1px solid gray',
-              borderRadius: '8px',
-              marginLeft: '20px',
-              padding: '7px'
-            }}
-            onClick={this._nextVerb}
-          >next >></p>
-        </div>
-        <div style={{ display: 'flex', width: '100%', flexWrap: 'wrap' }}>
-          {
-            Object.keys(tenses).filter(t => tenses[t]).map(tense => (
-              <div
-                key={`${tense}${verbs[idx]}`}
-                style={{
-                  margin: '20px',
-                  marginTop: '0px',
-                  border: '1px solid #aaa',
-                  borderRadius: '8px',
-                  padding: '15px'
-                }}
-              >
-              <div
-                key={`${tense}tense`}
-                style={{
-                  fontSize: '18px',
-                  fontWeight: '600',
-                  marginBottom: '7px'
-                }}
-              >{tense}</div>
-              <ConjugationForm
-                conjugations={
-                  verbs[idx].conjugations.find(
-                    v => v.tense.toLowerCase() == tense.toLowerCase()) || {} 
-                }
-                key={`${tense}CF`}
-                tense={tense}
-                verb={verbs[idx]}
-              />
+      <StyledContainer>
+        <TenseHeaderContainer>
+          <MediaQuery query="(max-device-width: 900px)">
+            <TenseHeaderMobile 
+              activeTenses={ tenses }
+              title={ 'Tense/Mood'}
+              tenseOptions={ Object.keys(tenses) }
+              toggleTense={ this._toggleTense}
```

Comment:
your spacing mismatches hurt me physically

Reply:
what are you on about here?

Reply:
this `{ something}`